src/components/list/list.mdx

```diff
@@ -8,23 +8,31 @@ menu: Components
 
 <limel-props name="limel-list" />
 
+## Custom styles
+
+| Class name | Description                                                                                     |
+| -------- | -------------------------------------------------------------------- |
+| `has-striped-rows` | Adds distinct styling and more interactivity to list items |
```

Comment:
The class name also does not indicate that it has any effect on the hovering. Should it still be in the same class or do you think it is a good use case to separate them?

Reply:
There may be a point in separating them actually, and this will increase our customization possibilities. On the other hand, I couldn't imagine a case where only one is needed, not the other one. But it can happen. Not really sure how to decide here. What do you think?

Reply:
Hmm I'm not sure :thinking: 
I think we should probably change the class name if we keep them together, but I'm not sure what the name should be then

Reply:
I think they should be split. I imagine not all lists will have items that do something when clicked, and then I assume they shouldn't zoom when hovered? But you might still want to have a striped list, I imagine. So I'd go with something like `has-striped-rows` and `has-interactive-items`